### DISCUSSION TITLE
Variable YAxis max insulin level

### DISCUSSION THREAD

**Discussion by vnniyB** [Navid200/xDrip#125] [Testing]
I've made a small change to the max insulin limit on the graph. I can manually set this value but I want to have it as a setting which I haven't been able to get working. 
https://github.com/NightscoutFoundation/xDrip/pull/1883/files

I'm not a Android/Java developer so would appreciate some guidance, please check out the commit on file to enable the Preference in the "Multiple Insulins" menu
`app/src/main/res/xml/xdrip_plus_prefs.xml`

**Comment by Navid200** [comment]
Why should entered insulin be shown in the graph with a vertical value?
I understand it has a time parameter, when you inject insulin.  Hence, showing it on the graph let's one see the time it was injected.
But, the vertical axis of the xDrip main screen is Glucose.  It's neither insulin nor carbs.   So, why would we need to represent the insulin value (or the carb value) by positioning the marker on the graph?

**Comment by tolot27** [comment]
@vnniyB Please add some screenshots for a better understanding what you are proposing.

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
